client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -63,12 +150,37 @@ const CancelPurchaseRefundInformation = ( { purchase } ) => {
 	}
 
 	return (
-		<p className="cancel-purchase__refund-information">{ text }</p>
+		<div className="cancel-purchase__info">
+			{ Array.isArray( text )
+				? text.map( ( paragraph, index ) =>
+					<p key={ purchase.id + '_refund_p_' + index } className="cancel-purchase__refund-information">
```

Comment:
Just curious, do we really need to specify `key` here? That's because you get a warning otherwise?